### PR TITLE
Add wix.vscode-import-cost

### PR DESCRIPTION
**Pull request by javavick**

### CODE REVIEW

**Comment by spoenemann** [comment]
Thanks for the contribution! Please add `"location": "packages/vscode-import-cost"` so the publishing script can find the correct location of the extension.

File: extensions.json [hunk 1]
```diff
@@ -940,6 +940,11 @@
       "id": "zxh404.vscode-proto3",
       "repository": "https://github.com/zxh0/vscode-proto3",
       "version": "0.4.2"
+    },
+    {
+      "id": "wix.vscode-import-cost",
+      "repository": "https://github.com/wix/import-cost",
+      "version": "2.12.0"
```
Comment (spoenemann) [inline extensions.json:947]:
```suggestion
      "version": "2.12.0",
      "location": "packages/vscode-import-cost"
```

**Review by spoenemann — APPROVED** [review]

**Comment by jankeromnes** [comment]
Update: Auto-publishing failed with this error:

```
Running: git clone --recurse-submodules https://github.com/wix/import-cost /tmp/repository
Running: npm install
Executing prepublish script 'npm run vscode:prepublish'...

> vscode-import-cost@2.12.0 vscode:prepublish /tmp/repository/packages/vscode-import-cost
> tslint {src,test}/**/*.ts && tsc -p ./

[FAIL] Could not process extension: {
  "id": "wix.vscode-import-cost",
  "repository": "https://github.com/wix/import-cost",
  "version": "2.12.0",
  "location": "packages/vscode-import-cost"
}
Error: Command failed: npm list --production --parseable --depth=99999 --loglevel=error
npm ERR! missing: import-cost@^1.8.0, required by vscode-import-cost@2.12.0
```

**Comment by spoenemann** [comment]
Probably we need to `npm install` in the project root first. But it seems `npm` doesn't support `--cwd` as `yarn` does, so I'd try to `cd` first.

**Comment by jankeromnes** [comment]
@spoenemann `npm install` is already executed in the project root:

https://github.com/open-vsx/publish-extensions/blob/25aaad9ce948fa4a4e993eeb06c1239d7ad383f1/publish-extensions.js#L110

However, maybe we also need to run it in `packages/vscode-import-cost`? E.g. with something like:

```js
  "prepublish": "cd packages/vscode-import-cost && npm install"
```

**Pull request merged**